src/main/java/org/dita/dost/ant/InitializeProjectTask.java

```diff
@@ -28,6 +28,11 @@
  * @since 3.5
  */
 public final class InitializeProjectTask extends Task {
+
+    private static ServiceLoader<StoreBuilder> storeBuilderLoader = ServiceLoader.load(StoreBuilder.class);
+
+    private String storeType = "file";
```

Comment:
Default `StoreBuilder` name, plain old temporary directory in disk implementation.